packs/data/equipment.db/flying-talon.json

```
@@ -0,0 +1,124 @@
+{
+    "_id": "ucz1WdBA0Ma34OsS",
+    "img": "systems/pf2e/icons/default-icons/weapon.svg",
+    "name": "Flying Talon",
+    "system": {
+        "MAP": {
+            "value": ""
+        },
+        "baseItem": "flying-talon",
+        "bonus": {
+            "value": 0
+        },
+        "bonusDamage": {
+            "value": 0
+        },
+        "category": "simple",
```

```suggestion
        "category": "advanced",
```
Advanced per James Jacobs